conductor.json

```diff
@@ -278,6 +301,16 @@
         "POSTGRES_PASSWORD": "{{env.DATABASE_PASSWORD}}",
         "POSTGRES_DATABASES": "'{{env.DATABASE_NAME}},{{env.DATABASE_NAME_TEST}},{{env.WORKFLOW_DATABASE_NAME}},{{env.WORKFLOW_DATABASE_NAME_TEST}}'"
       }
+    },
+    {
+      "name": "elasticsearch",
+      "image": "elasticsearch:7.8.0",
+      "port": 9200,
+      "env": {
+        "node.name": "elasticsearch-dev",
+        "discovery.type": "single-node",
+        "ES_JAVA_OPTS": "-Xms512m -Xmx512m"
```

Comment:
WUT?

Reply:
Heap size, I've reduced it - I can't find a minimum amount but I've made it 128mb for now

Reply:
default is 1GB

Reply:
I was about to ask for a comment... but JSON.

😡